src/main/java/invtweaks/InvTweaksHandlerShortcuts.java

```diff
@@ -437,16 +437,18 @@ private void moveAll(ShortcutConfig shortcut, ItemStack stackToMatch) {
 
                     // Continue if movement succeeded, there is another slot to try, or we're dropping items.
                     // In reverse: fail if movement failed, AND there are no other slots AND we're not dropping.
-                    if(success || (newIndex != toIndex) || (shortcut.action == ShortcutSpecification.Action.DROP)) {
+                    if (newIndex == toIndex) {
+                        break;
+                    } else if (success || shortcut.action == ShortcutSpecification.Action.DROP) {
                         toIndex = newIndex;
                     } else {
                         toIndex = -1;
                     }
                 }
             }
-            if(toIndex == -1) {
-                break;
-            }
+//            if(toIndex == -1) {
+//                break;
+//            }
```

Comment:
my fault, this patch is solve a problem in this condition:
Item with some specific tag on is and move it by using move all same item. than the game freeze.
cauing by the if condition is always true.